### PR TITLE
Mq shiree

### PR DESCRIPTION
**Pull request by ShireeWill**

### CODE REVIEW

File: src/_codux/boards/components/cart/cart.board.tsx [hunk 1]
```diff
@@ -16,7 +16,5 @@ export default createBoard({
     ),
     isSnippet: false,
     environmentProps: {
-        canvasHeight: 600,
```
Comment (alissawix) [inline src/_codux/boards/components/cart/cart.board.tsx:19]: I guess we should remove this

File: src/_codux/boards/image-test/image-test.board.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { createBoard } from '@wixc3/react-board';
```
Comment (alissawix) [inline src/_codux/boards/image-test/image-test.board.tsx:1]: what is that?

File: src/components/image-test/image-test.module.scss
Comment (alissawix) [inline src/components/image-test/image-test.module.scss:1]: and that :)

File: src/components/image-test/image-test.tsx
Comment (alissawix) [inline src/components/image-test/image-test.tsx:1]: and that :)

File: src/components/product-card/product-card.module.scss [hunk 3]
```diff
@@ -35,7 +35,7 @@
     }
 }
 
-@media (420px <=width < 980px) {
+@media (430px <=width < 980px) {
```
Comment (alissawix) [inline src/components/product-card/product-card.module.scss:38]: this should be the default, why media query and why up to 980?

File: src/pages/home-page/hero-image/hero-image.module.scss [hunk 4]
```diff
@@ -23,11 +25,10 @@
 .image {
     object-fit: cover;
     object-position: 50% 0%;
-    height: 640px;
-    width: 100vw;
+    // height: 640px;
```
Comment (alissawix) [inline src/pages/home-page/hero-image/hero-image.module.scss:28]: remove comment

File: src/pages/home-page/hero-image/hero-image.module.scss [hunk 5]
```diff
@@ -70,44 +75,75 @@
...
+    .big-title {
+        font-size: 85px;
+    }
+
+    .overlay-button {
+        height: 47px;
+    }
 }
 
 @media (980px <=width < 1400px) {
```
Comment (alissawix) [inline src/pages/home-page/hero-image/hero-image.module.scss:92]: that is a very strange media query
and something should be the default

File: src/pages/home-page/home-page.module.scss [hunk 6]
```diff
@@ -115,7 +115,7 @@
 }
 */
 
-@media (420px <=width < 980px) {
+@media (430px <=width < 980px) {
```
Comment (alissawix) [inline src/pages/home-page/home-page.module.scss:118]: same

File: src/_codux/boards/image-test/image-test.board.tsx (hunk 2 above)
Comment (ShireeWill) [reply src/_codux/boards/image-test/image-test.board.tsx:1]: It was a board to check something, will delete

File: src/pages/home-page/hero-image/hero-image.module.scss (hunk 5 above)
Comment (ShireeWill) [reply src/pages/home-page/hero-image/hero-image.module.scss:92]: Same, it's still in process.

File: src/_codux/boards/components/hero-image/hero-image.board.tsx [hunk 7]
```diff
@@ -1,5 +1,6 @@
 import { createBoard } from '@wixc3/react-board';
 import { HeroImage } from '../../../../pages/home-page/hero-image/hero-image';
+import imgSrc from '../../../../assets/img/[420_340]_heroImage04.jpg';
```
Comment (alissawix) [inline src/_codux/boards/components/hero-image/hero-image.board.tsx:3]: it isn't in use - should be deleted

**Review by alissawix — APPROVED** [review]

**Pull request merged**